azure-pipelines.yml

```diff
@@ -4,148 +4,170 @@
 # https://docs.microsoft.com/azure/devops/pipelines/languages/python
```

Comment:
Why are these changes being made?

Reply:
No idea! This was the result of merging with your commits. It had some conflicts there and I accepted the incoming change.

Reply:
I figured these were the changes related to running linting only once.

Reply:
I've pushed to this branch and reverted this back.